### ISSUE TITLE
Address Book does not save contacts

### ISSUE THREAD

**Issue by amrd24** [webtv-redialed/redialed-nodejs#8]
![image](https://github.com/user-attachments/assets/feb5e50e-6293-49b4-a2b4-a902bae7a26b)

**Comment by HIDEN64** [comment]
Odd. It doesn't seem to be sending any address headers.
```
* Request for wtv-mail:/addressbook from WebTV SSID <no> on 7914
 * Incoming headers on socket ID 7914 {
  request: 'POST wtv-mail:/addressbook',
  request_url: 'wtv-mail:/addressbook',
  Referer: 'wtv-mail:/addressbook?camefrom=inbox&action=edit&new_address=true&selected=new',
  'wtv-show-time': '29',
  pragma: 'no-cache',
  'wtv-request-type': 'primary',
  'wtv-system-cpuspeed': '166187148',
  'wtv-system-sysconfig': '4163328',
  'wtv-disk-size': '8006',
  'wtv-viewer': 'WebTVIntel--2.5',
  'wtv-incarnation': '66',
  'Content-type': 'application/x-www-form-urlencoded',
  'Content-length': '51',
  post_data: '6e69636b6e616d653d466f6f2b4261722663616d6566726f6d3d696e626f7826616374696f6e3d61646426646f6e653d416464',
  query: {
    nickname: 'Foo Bar',
    camefrom: 'inbox',
    action: 'add',
    done: 'Add'
  }
}
 * Found /home/dylan/Projects/redialed-service/ServiceVault/wtv-mail/addressbook.js to handle request (JS Interpreter mode) [Socket 7914]
/home/dylan/Projects/redialed-service/ServiceVault/wtv-mail/addressbook.js:703
                        var address = request_headers.query.address.split("@")[0];
                                                                    ^

TypeError: Cannot read properties of undefined (reading 'split')
    at /home/dylan/Projects/redialed-service/ServiceVault/wtv-mail/addressbook.js:703:48
    at sigintHandlersWrap (node:vm:279:12)
    at Script.runInContext (node:vm:146:14)
    at Script.runInNewContext (node:vm:153:17)
    at runScriptInVM (/home/dylan/Projects/redialed-service/app.js:363:18)
    at /home/dylan/Projects/redialed-service/app.js:707:37
    at Array.forEach (<anonymous>)
    at processPath (/home/dylan/Projects/redialed-service/app.js:436:24)
    at processURL (/home/dylan/Projects/redialed-service/app.js:1182:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
 * doErrorPage Called: 400 WebTV ran into a technical problem. Please try again.
 * Scripting or Data error: Headers were not defined. (headers,data) as follows:
7914 400 WebTV ran into a technical problem. Please try again.
Content-Type: text/html
 WebTV ran into a technical problem. Please try again.
 ```

**Comment by amrd24** [comment]
fixed in #10 due to a typo
